cmake/FindCSV.cmake

```diff
@@ -0,0 +1,31 @@
+#===============================================================================
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+########################################
+# Find libcsv
+include(GzPkgConfig)
+gz_pkg_check_modules_quiet(CSV libcsv)
```

Comment:
While I know for a fact no `pkg-config` module is (currently) released with `libcsv-dev`, `gz_manual_search` appears to rely internally on an earlier invocation of `gz_pkg_check_modules_quiet`. That's why this is here.